plugins/aws/resoto_plugin_aws/configuration.py

```diff
@@ -240,6 +240,10 @@ class AwsConfig:
             "If false, the error is logged and the resource is skipped."
         },
     )
+    collect_usage_metrics: bool = field(
```

Comment:
Should be either optional or you need a migration. Existing installations do not have this property.

Reply:
won't default value be sufficient here?